api/v1/helpers/verbs/findUtils.js

```diff
@@ -114,7 +126,11 @@ function toSequelizeWhere(filter, props) {
        */
       if (Array.isArray(props.fieldsWithEnum) &&
         props.fieldsWithEnum.indexOf(key) > -ONE) {
-        const enumArr = filter[key];
+
+        // if specified in props, convert the array in query to camelcase.
+        const enumArr = (props.fieldsToCamelCase &&
+          props.fieldsToCamelCase.indexOf(key) > -ONE) ?
+          convertArrayElementsToCamelCase(filter[key]) : filter[key];
 
```

Comment:
this is a ternary operator correct? What happens when 

props.fieldsToCamelCase && props.fieldsToCamelCase.indexOf(key) > -ONE is false ?